src/schema/objects/metadata.yaml

```diff
@@ -404,6 +404,8 @@ ChunkTransformationMatrix:
     scaling along the second and third axis respectively).
     Note that non-spatial dimensions like time and channel are not included in the
     transformation matrix.
+    In case of MRI data, the ChunkTransformationMatrix corresponds to the NIFTI [qform matrix]
+    (https://nifti.nimh.nih.gov/nifti-1/documentation/nifti1fields/nifti1fields_pages/qsform.html)
```

Comment:
Currently, this is only going to show up in the microscopy section, which does not seem very helfpul. If we want to this to be applicable to MRI data, then we need to add it to their sidecars.

I'm also unclear on what this means. Does this mean that ChunkTransformationMatrix must be equal to the the qform? Or that the qform is used and therefore `ChunkTransformationMatrix` does not apply to NIfTI images? (If so, then we should just remove this addition.)

Does this need a rule, such that `qform_code != 0` (or `== 1`)? What about `sform`? Can that be anything, or must that be identical to the `qform`, or have `sform_code == 0`?

Reply:
Agreed with @effigies, there is no need to duplicate the already existing information from the qform. So in the case of MRI, I would suggest to have this value set to `n/a` maybe?

Reply:
ChunkTransformationMatrix already isn't defined for MRI, so no need to even call it `n/a`. It can just be absent.

Reply:
Okay! I removed the MRI paragraph from the ChunkTransformationMatrix description in https://github.com/bids-standard/bids-specification/commit/d1e4f7c8a2b5d91c56805384a7e9e5f805257438.